src/less/dialog/ds6/dialog-base.less

```diff
@@ -0,0 +1,201 @@
+// MIXINS
+//-----------------------------
+
+// Element width in percentage - (No of gutters * gutter width / number of elements)px
+.dialog-grid(@cols, @gutter, @min, @max, @default: @min) {
+    width: calc(round(@default / @cols * 100%) ~"-" round((@default - 1) * @gutter / @default));
```

Comment:
`~"-"` ... cool trick!

Reply:
😎

Reply:
Inside a LESS file, if you use `calc()` with a `+` or `-` operator LESS treats the expression inside the parenthesis as something LESS needs to parse. Hence, LESS parses this:

```
calc(100% + 16px);
```

but will ignore this (using the `~""` as a way to escape the LESS compilation):

```
calc(100% ~"+" 16px);
```

-----

_P.S.:_ I found this out the hard way back in my C2C days. I ended up just escaping the whole `calc()` function like so:

```
~"calc(100% + 16px)";
```

Reply:
But, because Dylan is using LESS variables, he can't escape the entire `calc()`. He has to just escape the "triggering characters": `+` and `-`.